test/hiccup/compiler_test.clj

```diff
@@ -0,0 +1,13 @@
+(ns hiccup.compiler-test
+  (:require [hiccup.compiler :refer [render-element]]
+            [clojure.test :refer :all]))
+
+(deftest render-element-test
```

Comment:
Don't test the compiler functions directly; do it through the public `html` macro in the `core-test` namespace.